travis-autobuild-extensions.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ "$TRAVIS_REPO_SLUG" == "new-xkit/XKit" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
+  echo -e "Rebuilding extension dist...\n" &&\
+
+  cd $HOME &&\
+  git config --global user.email "travis@travis-ci.org" &&\
+  git config --global user.name "travis-ci" &&\
+  git clone https://"$GH_TOKEN"@github.com/new-xkit/XKit &&\
+  cd XKit &&\
+  git checkout gh-pages &&\
+  git merge master -m "Merge master $TRAVIS_COMMIT" &&\
+  npm install -g gulp &&\
```

Comment:
The gulp we installed for testing doesn't work, after_success already clears the entire container. That's also why I clone and npm install again. Otherwise it told me that gulp hasn't been found and I should run npm install gulp